packaging/installer/functions.sh

```diff
@@ -727,3 +727,64 @@ safe_sha256sum() {
 		fatal "I could not find a suitable checksum binary to use"
 	fi
 }
+
+install_or_remove_netdata_updater() {
```

Comment:
I'm not fond of these kinds of functions that 'decide' on their own what to do. I may want to change the logic. I may want to do something non-standard. If you want to have this kind of functionality you should — at the very least — create functions that `remove` and `install` and then invoke them from your `do_this_or_not_this_function()`. This way, you can easily compartmentalize the actual functionality from whatever logic you're using to manage your updater install.

Reply:
+1 on this